public/index.html

```diff
@@ -25,6 +25,7 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>React App</title>
+    <script>window.user = "{{token}}"</script>
```

Comment:
When this file gets templated, it's rendered as an embedded string. So `token` is a JSON string here:

```
window.user = "{\"key\": \"value\"}"
```

which will get rendered as a string instead of a JSON object. Instead:

```
window.user = {{token}}
```